django-stubs/contrib/auth/forms.pyi

```diff
@@ -114,28 +112,26 @@ class PasswordResetForm(forms.Form):
         extra_email_context: dict[str, str] | None = ...,
     ) -> None: ...
 
-class SetPasswordForm(forms.Form):
-    error_messages: _ErrorMessagesDict
+class SetPasswordForm(Generic[_UserType], SetPasswordMixin, forms.Form):
```

Comment:
This is not generic at runtime, which prevents this from being subclassed.

```python
class MySetPasswordForm(SetPasswordForm):
    pass
```

mypy complains: `error: Missing type parameters for generic type "SetPasswordForm"  [type-arg]`

```python
class MySetPasswordForm(SetPasswordForm[User]):
    pass
```

Fails at runtime: `TypeError: 'DeclarativeFieldsMetaclass' object is not subscriptable`

Reply:
@andersk can you please send a PR to `ext/`? So we can monkeypatch this object in runtime.

Reply:
Maybe we should add a small test to ensure that ? Something parsing pyi file for ``generic` calls and ensuring they match the one declared in the ext ? I can try something like that

Reply:
Sure, this would be a great addition. I have no idea on how to actually do that :)
Maybe a `stubtest` extra check? 🤔 

I can open issue in mypy.

Reply:
Would be nice if stubtest can catch that indeed

Reply:
https://github.com/python/mypy/issues/19032